### DISCUSSION TITLE
How to restrict pull request approval

### DISCUSSION THREAD

**Discussion by balazsbotond** [community/community#22819] [New to GitHub]
<p>Someone who has recently made valuable contributions to one of my projects has approved a pull request coming from someone else and added a (slightly weird and easy to misunderstand) comment. This might have made it seem like this was an official approval (and official weird comment) from one of the maintainers.</p>
<p>I like this person and don’t want to discourage him from contributing, but I as the sole maintainer and owner of the repo, want to be the only one who is able to approve a PR.</p>
<p>Is there a config setting for this that I have missed? Or does someone who has ever contributed to the repo gain approval privileges?</p>
<p>EDIT: I understand that this person doesn’t have write access and his approval does not count towards mergeability, as the help text says. Still, I find it very weird that basically, random people can approve PRs.</p>

**Comment by kingthorin** [comment, marked as answer]
<p>Yes basically anyone can approve a PR, however that approval should have a different coloured icon.</p>
<p>Further as the repo owner you should be able to edit or delete the comment or even the approval for that matter.</p>
<p>Of course you can always add a follow-up comment clarifying that “John Doe” has no authority to approve PRs and while his input is appreciated you still feel items xyz are still outstanding (or whatever).</p>

**Comment by balazsbotond** [comment]
<p>Thank you. The follow-up comment is a good idea! Marking this as solved.</p>
